packages/web/src/public/locales/en/users.json

```diff
@@ -21,15 +21,19 @@
     "user_exists": "A user with email \"{{email}}\" already exists."
   },
   "first_name": "First Name",
+  "forgot_password": "Forgot Password?",
```

Comment:
Do we really need version with a question mark and one without? Why not combine them?

Reply:
Done.